### DISCUSSION TITLE
Provision / DataPipeline / Druid failed - release-5.1.0

### DISCUSSION THREAD

**Discussion by heungheung** [Sunbird-Ed/Community#406] [Installation] [answered]
Doc link 
https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build

error message:
```
TASK [zookeeper-upgrade : download and unarchive zookeeper binary] *************
fatal: [10.90.16.101]: FAILED! => {
    "changed": false
}

MSG:

Failure downloading https://downloads.apache.org/zookeeper/stable/apache-zookeeper-3.6.3-bin.tar.gz, HTTP Error 404: Not Found
```
zk 363 is no longer in stable folder
https://downloads.apache.org/zookeeper/
Ansible role default
https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/release-5.1.0/ansible/roles/zookeeper-upgrade/defaults/main.yml

workaround is to update the default values to 3.6.4. Someone will need to create a PR to update the DP release-5.1.0 branch

**Comment by Krishnaj20** [comment]
@gandham-santhosh - Pls check and respond to this query.

@chitranshu-keshav @vinukumar-vs @anandp504

**Reply by gandham-santhosh** [reply]
I think moving to minor version should not have impact. As 3.6.x is already EoL, we can also plan to move to 3.7.x if there is no impact
@anandp504 please share your thoughts

**Reply by gandham-santhosh** [reply]
@anandp504 @manjudr can you please check this?

**Reply by chitranshu-keshav** [reply]
@anandp504 @manjudr can you please get back on this

**Comment by heungheung** [comment, marked as answer]
just for the benefit to rest of the community, quick workaround is actually changing the common.yml of the private inventory. default is just default and can be overridden. Whoever need to do fresh install can just add   
zookeeper_url: https://archive.apache.org/dist/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3-bin.tar.gz  
in the common.yml of DataPipeline and proceed.

**Reply by Krishnaj20** [reply]
thanks @heungheung for the contribution.
